src/mame/machine/archimds.cpp

```diff
@@ -34,7 +34,7 @@
 #include "debugger.h"
 
 static const int page_sizes[4] = { 4096, 8192, 16384, 32768 };
-static const uint32_t pixel_rate[4] = { 8000000, 12000000, 16000000, 24000000};
+//static const uint32_t pixel_rate[4] = { 8000000, 12000000, 16000000, 24000000};
```

Comment:
This array is now a local in `acorn_vidc10_device::screen_dynamic_res_change` - should be safe to remove the commented one here.